lua5.3/lua-5.3.5.json

```diff
@@ -1,15 +1,11 @@
 {
     "name": "lua-5.3",
-    "no-autogen": true,
-    "make-args": [
-        "CFLAGS=-O2 -g -fPIC -DLUA_COMPAT_5_2 -DLUA_COMPAT_5_1 -DLUA_USE_DLOPEN",
-        "linux"
-    ],
-    "make-install-args": [
-        "INSTALL_TOP=${FLATPAK_DEST}",
-        "TO_LIB=liblua.so liblua.so.5.3 liblua.so.5.3.5"
-    ],
-    "post-install": [
+    "buildsystem": "simple",
+    "build-commands": [
```

Comment:
Why change the build system type?

Reply:
Because I could not use the variable `$CFLAGS` in `make-args`. Using the autotools build system also seems kind of hacky as LUA is not using autotools.

Reply:
Ok well add `-j $FLATPAK_BUILDER_N_JOBS` to the make invocations then.

Reply:
Done.